example-library/src/main/resources/application-example.yaml

```
@@ -13,7 +13,13 @@
 # limitations under the License.
 #
 
+# Not interpreted by any java code, simply used by reference in the population.resources section
+default:
+  rootId: ../Palisade-examples/resources/data/
```

Given I would expect this path to be a directory resource rather than a system resource. I would change this to 
```suggestion
  rootId: ../Palisade-examples/resources/
```